src/main/kotlin/no/nav/syfo/dialogmelding/status/domain/DialogmeldingStatus.kt

```diff
@@ -45,6 +46,18 @@ data class DialogmeldingStatus private constructor(
             }
         }
 
+        fun createFromDatabase(
```

Comment:
Her synes jeg det høres ut som at vi gjør et kall til databasen for å kunne opprette dette objektet, men det stemmer jo ikke. Kanskje den kan hete noe annet? 🤔